test/Microsoft.AspNetCore.SystemWebAdapters.SessionState.Framework.Tests/RemoteSession/GetWriteableSessionHandlerTests.cs

```diff
@@ -133,7 +135,9 @@ public async Task DisconnectedRequest()
         response.Verify(r => r.FlushAsync(), Times.Once);
 
         cts.Cancel();
-        await Assert.ThrowsAsync<TaskCanceledException>(() => task);
+
+        var exception = await Record.ExceptionAsync(() => task);
```

Comment:
Were these throwing exceptions before?

Reply:
They were (because of the cts.Cancel)

Reply:
Yep, looks like it. I see the exception if I turn on first chance exceptions in the debugger.